web/app/forms.py

```diff
@@ -14,3 +14,23 @@ class SearchPlayerForm(Form):
     player_name = StringField('Username', validators=[])
     aga_id = StringField('AGA id', validators=[Optional()])
     submit = SubmitField()
+
+class TournamentForm(Form):
+    event_name = StringField("Tornament Name", validators=[Required()])
+    # start_date = db.Column(db.DateTime)
+    venue = StringField("Venue", validators=[Required()])
+    director = StringField("Director", validators=[Required()])
+    pairing = StringField("Pairing", validators=[Required()])
+    rule_set = StringField("Rule set", validators=[Required()])
+    submit = SubmitField()
```

Comment:
Note: The tournament object is eventually going to be more complex. The two big things that we're going need are:
1) Players in the tournament
2) Games (results)

Can be added later.